### PR TITLE
[v7r1] only code updates for shell scripts

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

File: Core/scripts/dirac-cert-convert.sh [hunk 1]
```diff
@@ -17,56 +17,56 @@ function usage {
...
 
-export OPENSSL_CONF=$HOME/.globus
-GLOBUS=$HOME/.globus
+export OPENSSL_CONF="${HOME}"/.globus
+GLOBUS="${HOME}"/.globus
 USERCERT_P12_ORIG=$1
-USERCERT_P12=$GLOBUS/$(basename "$USERCERT_P12_ORIG")
-USERCERT_PEM=$GLOBUS/usercert.pem
-USERKEY_PEM=$GLOBUS/userkey.pem
+USERCERT_P12="${GLOBUS}"/$(basename "${USERCERT_P12}_ORIG")
```
Comment (chrisburr) [inline Core/scripts/dirac-cert-convert.sh:28]: Is this a typo?
```suggestion
USERCERT_P12="${GLOBUS}"/$(basename "${USERCERT_P12_ORIG}")
```

Comment (chrisburr) [reply Core/scripts/dirac-cert-convert.sh:28]: This applies to several instances of `USERCERT_P12_ORIG`

File: tests/CI/run_tests.sh [hunk 2]
```diff
@@ -32,12 +32,12 @@ if [ "$INSTALLTYPE" == "server" ]; then
     ERR=0
     for repo_path in "${TESTREPO[@]}"; do
         cp -r "${repo_path}/tests" "$WORKSPACE/ServerInstallDIR/$(basename "${repo_path}")"
-        if [ "$(basename "${repo_path}")" == "DIRAC" ]; then
+	if [[ "$(basename "${repo_path}")" == "DIRAC" ]]; then
```
Comment (chrisburr) [inline tests/CI/run_tests.sh:35]:
```suggestion
        if [[ "$(basename "${repo_path}")" == "DIRAC" ]]; then
```

File: Core/scripts/dirac-cert-convert.sh (hunk 1 above)
Comment (fstagni) [reply Core/scripts/dirac-cert-convert.sh:28]: Ouf, thanks, that was a bad replacement. Fixed.

File: tests/CI/run_tests.sh (hunk 2 above)
Comment (fstagni) [reply tests/CI/run_tests.sh:35]: Thanks, fixed.

**Pull request merged**